main.js

```diff
@@ -160,4 +160,127 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+const tvTypes = inventory.map((tv) => {
+  return tv.type
+})
+
+console.log(tvTypes)
+
+const soldOut = inventory.filter((arr) => {
+  if (arr.originalStock === arr.sold) {
+    return true
+  }
```

Comment:
Je zou dit wat kunnen versimpelen
```suggestion
  return arr.originalStock === arr.sold
  }
```